sources/packages/backend/apps/api/src/route-controllers/assessment/assessment.controller.service.ts

```diff
@@ -147,6 +143,8 @@ export class AssessmentControllerService {
 
     return {
       assessment: assessment.assessmentData,
+      noaApprovalStatus: assessment.noaApprovalStatus,
```

Comment:
I know it is existing code but just wondering about values in Assessment suiting the noaApprovalStatus. That too on the Declined status of it. I believe this is the confirmation of assessment status and I believe currently a student cannot decline an assessment, is it something coming up @andrewsignori-aot ?

Reply:
We have a tech debit to remove the ones that are not used: https://app.zenhub.com/workspaces/student-information-management-system-5fce9df5aa1b45000e937014/issues/bcgov/sims/578

Reply:
Yes. Exactly.